osf/metadata/serializers/datacite/datacite_tree_walker.py

```diff
@@ -432,3 +433,20 @@ def _get_resource_type_general(self, focus_iri):
             if isinstance(type_term, rdflib.URIRef) and type_term.startswith(DATACITE):
                 return without_namespace(type_term, DATACITE)
         return 'Text'
+
+    def _visit_verified_links(self, parent_el):
```

Comment:
Can you provide an example local run result of the metadata output, assume we have some fake links and resource types?